samples/contrib/aws-samples/mnist-kmeans-sagemaker/mnist-classification-pipeline.py

```diff
@@ -116,20 +205,20 @@ def mnist_classification(region='us-east-1',
         max_wait_time=hpo_max_wait_time,
         checkpoint_config=hpo_checkpoint_config,
         role=role_arn,
-    )
+    ).after(process)
```

Comment:
what does this help with ?

Reply:
Processing doesn't export any values that are explicitly used in this component. This just enforces sequential execution, otherwise HPO will attempt to run in parallel to the pre-processing.

Reply:
I wonder if it might be best to just remove HPO from this pipeline instead since we do have another sample for HPO which is a more relevant example anyway